tests/drivers/entropy/api/entropy_psa_crypto.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,entropy = &rng_psa;
+	};
+
+	rng_psa: entropy_psa_crypto {
+		compatible = "zephyr,psa-crypto-rng";
+		label = "entropy_psa_crypto";
```

Comment:
Remove this (it's actually redundant given the node name; device names default to node names if the label property is unset).